tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "compilerOptions": {
+    "outDir": "dist",
+    "allowJs": false,
+    "target": "es5"
```

Comment:
I'm tempted to target es6 instead of es5. The only browser left that needs es5 is IE11.

Reply:
For visibility: this convo has been taken to the ui/ux guild, once we have an official list of supported browsers we can potentially update this target to whatever works for that list. For now I'm leaving this at es5